{
  "Title": "Security update for python36",
  "Tracking": {
    "ID": "SUSE-SU-2024:3929-1",
    "Status": "Final",
    "Version": "1",
    "InitialReleaseDate": "2024-11-06T15:33:43Z",
    "CurrentReleaseDate": "2024-11-06T15:33:43Z",
    "RevisionHistory": [
      {
        "Number": "1",
        "Date": "2024-11-06T15:33:43Z",
        "Description": "current"
      }
    ]
  },
  "Notes": [
    {
      "Text": "Security update for python36",
      "Title": "Topic",
      "Type": "Summary"
    },
    {
      "Text": "This update for python36 fixes the following issues:\n\nSecurity fixes:\n\n- CVE-2024-9287: properly quote path names provided when creating a virtual environment (bsc#1232241)\n\nOther fixes:\n\n- Drop .pyc files from docdir for reproducible builds (bsc#1230906)\n",
      "Title": "Details",
      "Type": "General"
    },
    {
      "Text": "The CVRF data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
      "Title": "Terms of Use",
      "Type": "Legal Disclaimer"
    },
    {
      "Text": "SUSE-2024-3929,SUSE-SLE-SERVER-12-SP5-LTSS-EXTENDED-SECURITY-2024-3929",
      "Title": "Patchnames",
      "Type": "Details"
    }
  ],
  "ProductTree": {
    "Relationships": [
      {
        "ProductReference": "libpython3_6m1_0-3.6.15-70.1",
        "RelatesToProductReference": "SUSE Linux Enterprise Server 12 SP5 LTSS Extended Security",
        "RelationType": "Default Component Of"
      },
      {
        "ProductReference": "libpython3_6m1_0-32bit-3.6.15-70.1",
        "RelatesToProductReference": "SUSE Linux Enterprise Server 12 SP5 LTSS Extended Security",
        "RelationType": "Default Component Of"
      },
      {
        "ProductReference": "python36-3.6.15-70.1",
        "RelatesToProductReference": "SUSE Linux Enterprise Server 12 SP5 LTSS Extended Security",
        "RelationType": "Default Component Of"
      },
      {
        "ProductReference": "python36-base-3.6.15-70.1",
        "RelatesToProductReference": "SUSE Linux Enterprise Server 12 SP5 LTSS Extended Security",
        "RelationType": "Default Component Of"
      }
    ]
  },
  "References": [
    {
      "URL": "https://www.suse.com/support/update/announcement/2024/suse-su-20243929-1/",
      "Description": "Link for SUSE-SU-2024:3929-1"
    },
    {
      "URL": "https://lists.suse.com/pipermail/sle-security-updates/2024-November/019784.html",
      "Description": "E-Mail link for SUSE-SU-2024:3929-1"
    },
    {
      "URL": "https://www.suse.com/support/security/rating/",
      "Description": "SUSE Security Ratings"
    },
    {
      "URL": "https://bugzilla.suse.com/1230906",
      "Description": "SUSE Bug 1230906"
    },
    {
      "URL": "https://bugzilla.suse.com/1232241",
      "Description": "SUSE Bug 1232241"
    },
    {
      "URL": "https://www.suse.com/security/cve/CVE-2024-9287/",
      "Description": "SUSE CVE CVE-2024-9287 page"
    }
  ],
  "Vulnerabilities": [
    {
      "CVE": "CVE-2024-9287",
      "Description": "A vulnerability has been found in the CPython `venv` module and CLI where path names provided when creating a virtual environment were not quoted properly, allowing the creator to inject commands into virtual environment \"activation\" scripts (ie \"source venv/bin/activate\"). This means that attacker-controlled virtual environments are able to run commands when the virtual environment is activated. Virtual environments which are not created by an attacker or which aren't activated before being used (ie \"./venv/bin/python\") are not affected.",
      "Threats": [
        {
          "Type": "Impact",
          "Severity": "moderate"
        }
      ],
      "References": [
        {
          "URL": "https://www.suse.com/security/cve/CVE-2024-9287.html",
          "Description": "CVE-2024-9287"
        },
        {
          "URL": "https://bugzilla.suse.com/1232241",
          "Description": "SUSE Bug 1232241"
        }
      ],
      "ProductStatuses": [
        {
          "Type": "Fixed",
          "ProductID": [
            "SUSE Linux Enterprise Server 12 SP5 LTSS Extended Security:libpython3_6m1_0-3.6.15-70.1",
            "SUSE Linux Enterprise Server 12 SP5 LTSS Extended Security:libpython3_6m1_0-32bit-3.6.15-70.1",
            "SUSE Linux Enterprise Server 12 SP5 LTSS Extended Security:python36-3.6.15-70.1",
            "SUSE Linux Enterprise Server 12 SP5 LTSS Extended Security:python36-base-3.6.15-70.1"
          ]
        }
      ],
      "CVSSScoreSets": {}
    }
  ]
}